src/main/groovy/org/boozallen/plugins/jte/init/primitives/injectors/GlobalCollisionValidator.groovy

```diff
@@ -95,7 +95,7 @@ import org.jenkinsci.plugins.workflow.steps.StepDescriptor
         List<String> functionNames = StepDescriptor.all().collect { step ->
             step.getFunctionName()
         }
-        List<String> stepCollisions = primitivesByName.keySet().intersect(functionNames)
+        List<String> stepCollisions = primitivesByName.keySet().intersect(functionNames) as List
```

Comment:
I think the real issue here is that keySet().intersect( ... ) is expecting a set so it bombs when there is a non-empty `functionNames` : https://github.com/jenkinsci/templating-engine-plugin/issues/250